### DISCUSSION TITLE
How to make sense of United Health index file

### DISCUSSION THREAD

**Discussion by hungtantran** [CMSgov/price-transparency-guide#645] [Q&A]
Hi everyone,

I'm looking through index files from United Health e.g this file is for Goldman Sachs https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_Goldman-Sachs--Co-LLC_index.json

In the above Goldman Sachs index file, each reporting plan has a list of in_network files like below:

`
"reporting_plans": [
        {
          "plan_name": "PREFERRED PROVIDER",
          "plan_id": "135108880",
          "plan_id_type": "EIN",
          "plan_market_type": "group"
        }
      ],
      "in_network_files": [
        {
          "description": "in-network files",
          "location": "https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_PPO-00_P3_in-network-rates.json.gz"
        },
        {
          "description": "in-network files",
          "location": "https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_Goldman-Sachs_CSP-900-T073_in-network-rates.json.gz"
        },
        {
          "description": "in-network CMC Cancer shared files",
          "location": "https://saticprod.blob.core.windows.net/optumodd/CMC_CRS_MRRF.json"
        },
        {
          "description": "in-network files",
          "location": "https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_PPO---NDC_PPO-NDC_in-network-rates.json.gz"
        },
        {
          "description": "in-network files",
          "location": "https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_Goldman-Sachs_CSP-954-C533_in-network-rates.json.gz"
        },
        {
          "description": "in-network CMC Transplant shared files",
          "location": "https://saticprod.blob.core.windows.net/optumodd/CMC_Transplant_MRRF.json"
        },
        {
          "description": "in-network files",
          "location": "https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_OBPM---Optum-Bundle-Payment_OBPM_in-network-rates.json.gz"
        }
      ],
`

I'm having trouble understanding what each of these in network file represents, how they are different from each other. For example, what is the difference between `2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_PPO-00_P3_in-network-rates.json.gz` vs `2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_Goldman-Sachs_CSP-900-T073_in-network-rates.json.gz`.

Would really appreciate if anyone can help here. Thanks!

**Comment by bryanbriegel** [comment]
A cursory review suggests the PPO-xxx is a PPO network rate file, and the CSP - *may* be a Cancer Support Program rate file. ("That’s why we offer the UnitedHealthcare Cancer Support Program (CSP)" - a quote from a UHC site.   Thats an inference only.  PPO is more 'certain'.  This is something that likely needs to be confirmed directly with UHC.
